frontend/src/scenes/funnels/funnelLogic.ts

```diff
@@ -1258,7 +1258,7 @@ export const funnelLogic = kea<funnelLogicType<openPersonsModelProps>>({
                 return
             }
 
-            const funnelStep = converted ? step.order : -step.order
+            const funnelStep = converted ? step.order : -step.order - 1
```

Comment:
there's a similar line below, doing the same calculation again.

For dropoffs from (1-indexed) step 2 in the funnel, order = 1, and I'd put in... -2. Ok, this makes sense!